README.md

```diff
@@ -25,3 +25,14 @@ at your option.
 Unless you explicitly state otherwise, any contribution intentionally submitted
 for inclusion in the work by you, as defined in the Apache-2.0 license, shall be dual licensed as above, without any
 additional terms or conditions.
+
+##### BROKEN ON WINDOWS MSVC
+
+Download your Vendors OpenCL SDK:
```

Comment:
Users do not need to install the SDK in the majority of cases. The appropriate OpenCL drivers are often included with standard drivers. Again I'll need to double check the details on this and I'll help you make tweaks to this description.